charts/tracetest-onprem/values.yaml

```diff
@@ -90,33 +90,53 @@ global:
       path: "/"
 
 nats:
+  credentials:
+    username: admin
+
   enabled: true
-  
+
+  natsBox:
+    enabled: false
+
   config:
+    cluster:
+      enabled: true
+      replicas: 3
     jetstream:
       enabled: true
       fileStore:
-        enabled: true
-        dir: /data
         pvc:
-          enabled: true
           size: 10Gi
       memoryStore:
-          enabled: true
-          maxSize: 1Gi
+        enabled: true
+        maxSize: 1Gi
+    merge:
+      accounts:
+        $SYS:
+          users:
+          - {user: << $NATS_ADMIN_USERNAME >>, password: << $NATS_ADMIN_PASSWORD >>}
+
+  container:
+    merge:
+      env:
+      - name: NATS_ADMIN_USERNAME
+        valueFrom:
+          secretKeyRef:
+            name: tracetest-nats-credentials
+            key: username
+      - name: NATS_ADMIN_PASSWORD
+        valueFrom:
+          secretKeyRef:
+            name: tracetest-nats-credentials
+            key: password
+      - name: SERVER_NAME
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.name
```

Comment:
Is this being used somewhere?

Reply:
`$SERVER_NAME` is the default server_name config for nats. this setting makes the internal nats server name to match the pod name. this helps when debugging